unit-tests/dds/test-librs-device-properties.py

```diff
@@ -0,0 +1,124 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+from rspy import log, test
+log.nested = 'C  '
+
+import d435i
+import d405
+import d455
+
+import pyrealsense2 as rs
+rs.log_to_console( rs.log_severity.debug )
+from time import sleep
+
+context = rs.context( '{"dds-domain":123}' )
+only_sw_devices = int(rs.product_line.sw_only) | int(rs.product_line.any_intel)
+
+
+def wait_for_devices( tries = 3 ):
```

Comment:
I see this is implemented a few times.
Maybe move it to dds.py as a service func?

Reply:
Moved